.taskcluster.yml

```diff
@@ -87,38 +87,67 @@ tasks:
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
       - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
         then:
+          taskId: {$eval: as_slugid("build_release")}
           dependencies:
             - {$eval: as_slugid("lint_task")}
             - {$eval: as_slugid("test_task")}
-          scopes:
-            - secrets:get:project/relman/dump_syms/deploy
           created: {$fromNow: ''}
           deadline: {$fromNow: '2 hour'}
           provisionerId: proj-relman
           workerType: ci
           payload:
-            features:
-              taskclusterProxy:
-                true
             maxRunTime: 3600
             image: "rust:buster"
             env:
               PROJECT_NAME: dump_syms
             command:
               - "/bin/bash"
               - "-cx"
-              - "curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py &&
-                 python3 get-pip.py &&
-                 rustup toolchain install stable &&
-                 rustup default stable &&
-                 git clone --recursive --quiet ${repository} &&
-                 cd dump_syms &&
+              - "git clone --recursive --quiet ${repository} &&
+                 cd dump_syms && mkdir -p /build &&
                  git -c advice.detachedHead=false checkout ${head_rev} &&
-                 cargo build --release &&
-                 pip3 install -r infra/requirements.txt &&
-                 python3 infra/release.py"
+                 cargo build --release --target-dir=/build &&
+                 cd /build/release && tar -zvcf /build.tar.gz dump_syms"
```

Comment:
This will change the contents of the release asset, as currently there is a directory and dump_syms is inside the directory.
I don't know if there is something relying on the directory structure, @calixteman?

Reply:
The script Tom landed last week is using it:
https://hg.mozilla.org/mozilla-central/file/tip/taskcluster/docker/system-symbols-win/Dockerfile

Reply:
Ah right. FYI, that's changing soon(ish) as we will use the dump_syms built in mozilla-central.